includes/post-types.php

```diff
@@ -66,7 +66,7 @@ function give_setup_post_types() {
 			'not_found_in_trash' => __( 'No forms found in Trash.', 'give' ),
 			'parent_item_colon'  => '',
 			'menu_name'          => apply_filters( 'give_menu_name', __( 'Donations', 'give' ) ),
-			'name_admin_bar'     => apply_filters( 'give_name_admin_bar_name', __( 'Donation Form', 'give' ) ),
+			'name_admin_bar'     => apply_filters( 'give_name_admin_bar_name', __( 'Campaign', 'give' ) ),
```

Comment:
Let's just remove this from the Admin Bar for now, especially since this is hijacking the Donation Form post type definition in the legacy code. We can look at adding this at a later time as a feature of the Campaign domain.